mitoinstaller/mitoinstaller/log_utils.py

```diff
@@ -88,6 +89,29 @@ def log_error(event: str, params: Dict[str, Any]=None) -> None:
         }
     )
 
+experiment = None
+def _get_experiment_params() -> Dict[str, Any]:
+    """
+    Get data relevant for tracking the experiment, so we can 
+    see how the experiment is running
+
+    NOTE: this must match the function in mitosheet
+    """
+    global experiment
+    if experiment is None:
+        experiment = get_current_experiment()
+
+    if experiment is None:
+        experiment_params = {'experiment_id': 'No experiment'}
+    else:
+        experiment_params = {
+            'experiment_id': experiment["experiment_id"],
+            'experiment_variant': experiment["variant"],
+            f'experiment_{experiment["experiment_id"]}': experiment['variant']
```

Comment:
Do we purposefully have this information redundantly so we can create dashboards?

Reply:
Yep! We need to this to be in every log!

Reply:
Sorry I meant, the redundancy of having `experiment_id` `experiment_variant` and then the last item which is `experiment_id: experiment_variant`.